modules/ROOT/pages/scatter-gather-concept.adoc

```diff
@@ -76,7 +76,7 @@ Example of handling these errors:
         </route>
     </scatter-gather>
     <error-handler>
-        <on-error-continue type="COMPOSITE_ROUTING">
```

Comment:
COMPOSITE_ROUTING (without the MULE: prefix) does not produce an exception. Is it valid for this component?

Reply:
It's valid :) the `MULE` prefix is not necessary